src/Branching/effective-feature-isolation-on-tfvc.md

```diff
@@ -0,0 +1,113 @@
+
+> [!TIP]
+>
+> Keep your feature branch **short-lived**! 
+> 
+> Unlike main or other central branches, feature branches have a limited life time. They are scoped to features, bugs, and hot fixes, which are typically developed within a sprint or iteration. Consider removing your feature branch once your feature has met your team's definition of done [DoD](https://aka.ms/vsardod) and changes have been merged with the parent branch.
+
+As the number of your feature branches increase, your storage requirements and branch hierarchy visualization noise increases. With only five feature branches, the diagram is already noisy and rapidly losing its monitoring qualities. Can you imagine the impact of your team creating hundreds of feature branches?
+ 
+![Feature isolation noise](./_img/effective-feature-isolation-on-rfvc/feature-isolation-basic-noise.png)
+
+Similarly, the **Source Control Explorer** view becomes noisy and impractical as the number of branches increases. Unless you have a consistent naming convention, finding a specific feature branch among hundreds of feature branches is challenging.
+
+![Feature isolation noise](./_img/effective-feature-isolation-on-rfvc/feature-isolation-basic-noise-vside.png)
+
+> [!TIP]
+>
+> **Delete** your feature branch when you're done!
+
+#Impact of deleting a feature branch
+
+By [deleting](https://www.visualstudio.com/en-us/docs/tfvc/delete-restore-files-folders#delete-an-item) feature branches you minimise noise and focus on active feature development. 
+
+![Delete feature branches](./_img/effective-feature-isolation-on-rfvc/feature-isolation-delete-branches.png)
+
+Remember it's a soft delete and your history is not lost. You can show deleted branches as follows:
+
+- Select **Tools > Options > Visual Studio Team Foundation Server**.
+- From the submenu, select **Show deleted items in the Source Control Explorer** to view the deleted branch(es). 
+
+If needed, you can also [restore](https://www.visualstudio.com/en-us/docs/tfvc/delete-restore-files-folders#restore-items-deleted-from-visual-studio) deleted branches and associated items. 
+
+![Show deleted feature branches](./_img/effective-feature-isolation-on-rfvc/feature-isolation-delete-branches-show.png)
+
+> [!NOTE]
+>
+> Use the [destroy](https://www.visualstudio.com/en-us/docs/tfvc/destroy-command-team-foundation-version-control) command with caution. It's a **permanent** delete!
+
+If no one in your team has destroyed a branch, you'll have the complete history needed for auditing and migration tools that rely on replaying history.
+
+So, by keeping your branches **short-lived** and adopting a **consistent naming convention**, the feature isolation strategy is effective for small and large teams.
```

Comment:
Then again we can add a note for user to consider feature toggles ad an alternative or complementary strategy. @rhundhausen thanks for all your feedback!